courses/machine_learning/deepdive2/feature_engineering/solutions/4_keras_adv_feat_eng.ipynb

```diff
@@ -0,0 +1,1751 @@
+{
```

Comment:
<p>I'd remove the bold here and just make a comment to check. And in the next cell show the output of </p><p>!head data/*.csv</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>